MiniBootcamp/MiniBootcamp/TweetFeed/TweetCell/TweetCellViewModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  TweetCellViewModel.swift
+//  MiniBootcamp
+//
+//  Created by Edgar Solis on 27/05/23.
+//
+
+import UIKit
+
+struct TweetCellViewModel {
+    let userName: String
```

Comment:
After declaring either a class, a struct or an enum, you have to leave and empty line. 

```suggestion
struct TweetCellViewModel {

    let userName: String
```